files/pt-br/web/api/window/applicationcache/index.md

```diff
@@ -21,7 +21,11 @@ cache = window.applicationCache
 
 ## Especificação
 
-- {{spec("https://www.w3.org/TR/2008/WD-html5-20080122/#appcache","HTML 5","WD")}}
+{{Specifications}}
+
```

Comment:
No specification table in en-US:

https://github.com/mdn/content/blob/2333e01ceb47d178b9e8fc9efeff7da97a47cf49/files/en-us/web/api/window/applicationcache/index.md?plain=1#L25-L29